app/views/internal/broadcasts/index.html.erb

```diff
@@ -31,6 +31,7 @@
           <p>
             <%= broadcast.processed_html %>
           </p>
+          <h3>Last Active: <%= broadcast.last_active_at %></h3>
```

Comment:
Should I format this timestamp at all? 🤔

Reply:
Yep, we definitely should; the current format isn't really human-readable. Check out Ruby's `strftime` [method](https://apidock.com/ruby/DateTime/strftimehttps://github.com/juliannatetreault/dev.to/tree/juliannatetreault/last-active-broadcast).

Reply:
Agreed! I have since changed this and followed the convention of some other formatted times (podcasts and events), but I could also remove the year from the timestamp if we think that it isn't necessary.